pc/segmentation/regiongrowing/regiongrowing_test.go

```diff
@@ -26,28 +26,28 @@ func TestRegionGrowingSegment(t *testing.T) {
 		return points
 	}
 
-	objects := map[string]*struct {
+	objects := []*struct {
 		pos    mat.Vec3
 		points pc.Vec3Slice
 		label  uint32
 		indice []int
 	}{
-		"floor": {
+		{ // floor
```

Comment:
```go
const (
  objectFloor = iota
  objectBox1
  ...
)

objects := []*struct {
  ...
}{
  objectFloor: {
```
and 
```go
		"Label0": {
			p:        mat.Vec3{0.5, 0.1, 0},
			maxRange: 0.15,
			indice:   objects[objectFloor].indice,
		},
```

or sorting the result before comparing would be more straightforward.

Reply:
done in 2d39c72dc062511ea757085fc413e08a1d55e747